README.md

```diff
@@ -3,50 +3,76 @@
 [![Build Status](https://api.travis-ci.org/b9org/b9.svg?branch=master)](https://travis-ci.org/b9org/b9)
 [![Coverage Status](https://coveralls.io/repos/github/b9org/b9/badge.svg?branch=master)](https://coveralls.io/github/b9org/b9?branch=master)
 
-Base9 is an educational language runtime and programming language! We’re using it to show people how to use
-[Eclipse OMR] to build their own language runtime with a Just in Time (JIT) Compiler! The Base9 language, b9porcelain, is a simple subset of JavaScript.
+b9 is an educational language runtime and programming language! We’re using it to show people how to use
+[Eclipse OMR] to build their own language runtime with a Just in Time (JIT) Compiler! The b9 language, b9porcelain, is a simple subset of JavaScript.
 
 [Eclipse OMR]: https://github.com/eclipse/omr
 
-## Requirements
+This page contains basic instructions on building b9 in an Ubuntu 16.04 environment.
+You may copy each of the blocks of commands and paste them in the terminal.
+Depending on your user privileges, you may need to precede some of commands with
+sudo in steps 1 and 3. For more detailed instructions, check out our 
+[set-up page](./docs/SetupBase9.md).
 
-To get started with Base9, you'll need:
+### 1. Requirements
 
-`git, build-essential, nodejs, npm, esprima, cmake, ninja` 
+To get started with b9 on Ubuntu 16.04, you'll need the following:
 
-For detailed instructions on setting up Base9, check out our [set-up page](./docs/SetupBase9.md).
+* `git` 
+* `build-essential`
+* `nodejs`
+* `npm` 
+* `cmake` 
+* `ninja`
 
+To install the applications above, run the following command
+```plaintext
+apt-get update && apt-get install git build-essential nodejs npm cmake ninja
+```
 
-## Clone the repository and get the submodules
+### 2. Clone the repository and get the submodules
 
 ```
-git clone --recursive https://github.com/b9org/b9.git
-cd Base9
-git submodule update --init
+git clone --recursive https://github.com/b9org/b9.git \
+&& cd b9 \
+&& git submodule update --init \
+&& cd ..
+```
+
+### 3. Upgrade and create link for nodejs and install esprima
+
+```plaintext
+npm install -g n \
+&& n latest \
+&& ln -s /usr/local/bin/node /usr/bin/node \
```

Comment:
I am not crazy about using `n`. There are many ways to obtain node binaries. I think documenting our minimum version of nodejs in the main readme is a better way to go.

If Ubuntu's default packaged nodejs binary really is too old, we can suggest `n` as one way to manage nodejs versions. But even then, we should elaborate a little more on `n`, and provide a link.

Did we ever figure out what our minimum nodejs version is?

Reply:
The minimum version needed is version 4.5 since `Buffer.alloc()` was added in that version. I will document that in the main Readme.

Reply:
OK that would be awesome, thanks!

Reply:
OK so, (as you've said) on ubuntu 16.04 LTS, the default version really is too old.

Maybe we should link people to [this](https://nodejs.org/en/download/package-manager/#debian-and-ubuntu-based-linux-distributions) document? This looks slightly more legit than `n`.